saphir/src/redirect.rs

```diff
@@ -32,6 +33,7 @@ pub struct Builder {
     form_data: Option<Result<HashMap<String, String>, BuilderError>>,
     content: Option<Box<dyn TransmuteBody + Send + Sync>>,
     content_type: Option<Mime>,
+    extra_headers: HashMap<String, String>,
```

Comment:
this field should be of type `HeaderMap<HeaderValue>` to be consistent with the `response::Builder`